tests/expr_and_series/dt/datetime_attributes_test.py

```diff
@@ -49,10 +50,15 @@ def test_datetime_attributes(
         request.applymarker(pytest.mark.xfail)
     if attribute == "date" and "cudf" in str(constructor):
         request.applymarker(pytest.mark.xfail)
-    if "pyspark" in str(constructor):
-        request.applymarker(pytest.mark.xfail)
 
-    df = nw.from_native(constructor(data))
+    if "pyspark" in str(constructor):
+        data_ = {
+            col_name: [v.replace(tzinfo=timezone.utc) for v in col_values]
+            for col_name, col_values in data.items()
+        }
```

Comment:
This is most likely needed for everyone not based in utc time. Since in `pyspark_lazy_constructor` we set `.config("spark.sql.session.timeZone", "UTC")`, the results will not be the same otherwise. 

Context: it took me a bit to realize why instead of hours 12 and 2 I was getting 11 and 1

Reply:
shall we just put the tzinfo to `timezone.utc` when we define `data=`?

Reply:
I tried, and all pandas like backends raise warnings/errors sadly